### DISCUSSION TITLE
http/https proxy to download white list

### DISCUSSION THREAD

**Discussion by smutel** [rspamd/rspamd#3624] [Q&A]
Hello,

I have this kind of errors in the web user interface:
error reading https://maps.rspamd.com/freemail/free.txt.zst(151.115.41.123:443): connection with http server terminated incorrectly: ssl connect error: syscall fail: Connection timed out

The server where rspamd does not have access to internet. But it could have access through a http/https proxy on port 3128.
Is-it possible to setup this in rspamd ?

Thanks.

**Comment by vstakhov** [comment]
As possible workaround, you can download these maps manually (e.g. by using cron + wget) and override the default maps paths from external HTTP resources to these local files.

**Reply by smutel** [reply]
Thank you for your answer. I did it for some part of the config. I also create a new issue https://github.com/rspamd/rspamd/issues/3674.

What about fuzzy check ? Is-it possible to do the same ? Or I need just to disable it for the moment ?
